components/Notifications/SubscribeCheckbox.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { Checkbox, mediaQueries } from '@project-r/styleguide'
+import { compose } from 'react-apollo'
+import {
+  withSubToDoc,
+  withUnsubFromDoc,
+  withSubToUser,
+  withUnsubFromUser
+} from './enhancers'
+import { css } from 'glamor'
+import withT from '../../lib/withT'
+
+const styles = {
+  checkbox: css({
+    display: 'inline-block',
+    width: '100%',
+    [mediaQueries.mUp]: {
+      width: '50%'
+    }
+  }),
+  checkboxCallout: css({
+    '& label': {
+      fontSize: 16,
+      display: 'flex',
+      textAlign: 'left',
+      alignItems: 'center'
+    }
+  }),
+  checkboxLabelCallout: css({
+    marginTop: -6
+  })
+}
+
+const SubscribeCheckbox = ({
+  t,
+  subToDoc,
+  unsubFromDoc,
+  subToUser,
+  unsubFromUser,
+  subscription,
+  setAnimate,
+  callout,
+  filters,
+  filterName
+}) => {
+  const isCurrentActive = filters
+    ? filters.includes(filterName) && subscription && subscription.active
+    : subscription && subscription.active
+  const activeFilters = (subscription.active && subscription.filters) || []
+  const isDocument =
+    subscription &&
+    subscription.object &&
+    subscription.object.__typename === 'Document'
+
+  const toggleCallback = () => setAnimate && setAnimate(true)
+
+  const toggleSubscribe = () => {
```

Comment:
we should handle loading and error of the mutation: add a new `isLoading`/`isMutating` state which sets ` <Checkbox disabled />` and a `serverError` state which sets `<Checkbox error />` and potential prints the message somewhere, but just marking the error is a good first step. You can get the error with a `.catch` after the `.then`.